doc/CONTRIBUTING.md

```diff
@@ -2,13 +2,14 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 # Contributing
 
-- [Pull Requests](#pull-requests)
-- [Requirements](#requirements)
-- [Development Workflow](#development-workflow)
-- [Build](#build)
-- [Structure](#structure)
-  - [VS Code Patch](#vs-code-patch)
-  - [Currently Known Issues](#currently-known-issues)
+- [Contributing](#contributing)
+  - [Pull Requests](#pull-requests)
+  - [Requirements](#requirements)
+  - [Development Workflow](#development-workflow)
+  - [Build](#build)
+  - [Structure](#structure)
+    - [VS Code Patch](#vs-code-patch)
```

Comment:
What do you think about calling this `VS Code Modifications` or `VS Code Changes` or something? Or `Modifications to VS Code`.

Reply:
Yeah, we should rename that! I like `Modification to VS Code`